src/css/index.scss

```diff
@@ -0,0 +1,103 @@
+.Info-text1{ 
```

Comment:
Usually we will have a space following the selector.

Reply:
Missing space between the class name and `{`, good code formatting is as important as the code quality.